.travis.yml

```diff
@@ -67,14 +69,16 @@ jobs:
       before_install:
         - sudo apt-get update -q
         - sudo apt-get install -y fakeroot
-      after_script:
+      script:
+        - mvn clean package -DskipTests # no need to run tests again
         - cd phoenicis-dist/src/scripts
         - bash phoenicis-create-package.sh
     - stage: Packages
       name: dmg
       os: osx
       osx_image: xcode10.1
-      after_script:
+      script:
+        - mvn clean package # run tests for OSX (doesn't run in 'build + test')
```

Comment:
Why do you not sue `-DskipTests` here?

Reply:
Read the comment.

Reply:
Why don't we run it in `build + test`?
Does it throw some kind of error?

Reply:
Yes, logically it makes sense but it would run the Maven build twice. Maybe open a PR with this then we can decide there.

Reply:
Yes, the same as we do for our linux builds :)

Reply:
yep (it was like this before for OSX as well)